docs/install.md

```diff
@@ -87,3 +87,6 @@ key = "/etc/pki/katello/private/pulp-client.key"
 verify_ssl = false
 ```
 
+As Katello uses Pulp as a backend all modifying actions in Pulp should be performed via Katello.
+Therefore you are also strongly encourged to set `dry_run = false`, to prevent accidentally calling into dangerous commands.
```

Comment:
Should that have been `dry_run = true`? ;-)

Reply:
absolutely!